### PR TITLE
MetaLamp/lending-pool/Configure nix for Lendng pool

### PR DESCRIPTION
**Pull request by olgaklimenko**
Nix configuration based on [plutus-starter](https://github.com/input-output-hk/plutus-starter).

### CODE REVIEW

File: MetaLamp/lending-pool/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```
Comment (olgaklimenko) [inline MetaLamp/lending-pool/package-lock.json]: @performanceArtist is it okay to push that file, or better to ignore it?

**Review by kelizarov — APPROVED** [review]
LGTM

**Review by stanislav-az — APPROVED** [review]

**Comment by stanislav-az** [comment]
Can you add `purty` to dependencies?
Also it would be very helpful to add `fix-stylish-haskell` and `fix-purty` commands from plutus

File: MetaLamp/lending-pool/package-lock.json (hunk 1 above)
Comment (performanceArtist) [reply MetaLamp/lending-pool/package-lock.json]: This file should be present in diff.

Comment (performanceArtist) [reply MetaLamp/lending-pool/package-lock.json]: Although wait - shouldn't it be in the client folder? Where did it come from? If you created it by mistake, it should be removed.

**Comment by olgaklimenko** [comment]
> fix-purty



> Can you add `purty` to dependencies?
> Also it would be very helpful to add `fix-stylish-haskell` and `fix-purty` commands from plutus

Done.

**Comment by olgaklimenko** [comment]
@stanislav-az You can add by yourself all dependencies, which can be inherited from `plutus` using that [commit](https://github.com/input-output-hk/plutus-use-cases/pull/60/commits/26563e5136e89ead767600562a94024b0dd709ec) as example.

File: MetaLamp/lending-pool/package-lock.json (hunk 1 above)
Comment (olgaklimenko) [reply MetaLamp/lending-pool/package-lock.json]: Now I see, that file appeared from mistaken `npm install` call from `lending pool` directory, thanks

Comment (olgaklimenko) [reply MetaLamp/lending-pool/package-lock.json]: removed

**Review by performanceArtist — APPROVED** [review]
Everything seems to work.

**Pull request merged**